packages/flutter/lib/src/material/date_picker_theme.dart

```diff
@@ -111,7 +111,7 @@ class DatePickerThemeData with Diagnosticable {
   ///
   /// The dialog's header displays the currently selected date.
   ///
-  /// The color of the [headerHeadlineStyle] is not used directly, as
+  /// The [TextStyle.color] of the [headerHeadlineStyle] is not used directly,
```

Comment:
Is it not used directly, or is it just not used?

Reply:
Sorry, I should have simplified that to just "not used".